app/Http/Controllers/Dashboard/IncidentController.php

```diff
@@ -321,6 +321,10 @@ public function createIncidentUpdateAction(Incident $incident)
                 ->withErrors($e->getMessageBag());
         }
 
+        if ($incident->incident->component) {
+            $incident->incident->component->update(['status' => Binput::get('component_status')]);
```

Comment:
The component and the status should be passed through to the `CreateIncidentUpdateCommand` :)

Reply:
noted, updated my commit.

Reply:
These lines need moving into the `CreateIncidentUpdateCommandHandler`.